Header header
int32 self_drone_id
node_detected[] detected_nodes